README.md

```diff
@@ -1 +1,121 @@
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [📝 License](#license)
+
+
+# 📖 [Portfolio mobile project] <a name="about-project"></a>
+
+**[Portfolio mobile project]** is a project that describe my skills and previous work experience.
+
+## 🛠 Built With <a name="built-with">html and css</a>
+
+### Key Features <a name="key-features"></a>
+
+- **[contact form ]**
+- **[projects show cards]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://eyoab01.github.io/tesfaye-capstone-project/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+1.copy the repository link from github.(https://github.com/eyoab01/esfaye-capstone-project)
+2.open vscode on your computer clone it to your new folder.
+3.wright click on the index.html file and select open with live server.
+
+### Prerequisites
+
+In order to run this project you need:
+vscode editor (you can download it from https://code.visualstudio.com/download)
+
+### Setup
+
+create a folder and name it 'my-folder'
+open with vscode editor > right click on the folder and select >open in integrated terminal option
+cd my-folder
+git clone https://github.com/eyoab01/tesfaye-capstone-project.git
+
+### Usage
+
+To run the project, do the following:
+go to your project folder and right click on the index.html file and select open with live server.
+
+### Run tests
+
+To run tests, do the following step:
+go to your project folder and right click on the index.html or https://eyoab01.github.io/tesfaye-capstone-project/
+
+### Deployment
+
+You can deploy this project using:
+vscode or github.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Tesfaye Teklu**
+
+- GitHub: [@eyoab01](https://github.com/eyoab01)
+- Twitter: [@tesfayeteklu10](https://twitter.com/@tesfayeteklu10)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/tesfaye-teklu-352067105)
+- website: [https://eyoab01.github.io/mobile-portfolio-project-final/]
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/eyoab01/tesfaye-capstone-project/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project you can support any time.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
```

Comment:
- [ ] Kindly make sure the README contains an "Acknowledgements" section or similar that gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).